chapter_1/exercise_1_22/fold_line.c

```diff
@@ -43,30 +43,44 @@ int get_line(char line[], int max_line_len)
 
 void fold_line(char line[], char fold_str[], int n_break)
 {
-  int i, j;
+  int i, j, m;
+  int column = 0;
   int split = FALSE;
   int offset = 10;

```

Comment:
@illustrofia I think the best approach for this functionality is to move the offset variable into an OFFSET define.